lib/macho/macho_file.rb

```diff
@@ -424,14 +478,24 @@ def low_fileoff
       offset
     end
 
+    # Updates the number of load commands in the raw data.
+    # @param ncmds [Fixnum] the new number of commands
+    # @return [void]
+    # @private
+    def set_ncmds(ncmds)
```

Comment:
Actually, let's just leave it as-is and change this later (if it causes any confusion). I prefer to get this merged now so that we can move on with other tasks that are high priority. (It's not like this is inherently bad and it's consistent with the current code.)